test/images/apparmor-loader/loader.go

```diff
@@ -17,19 +17,18 @@ limitations under the License.
 package main
 
 import (
-	"bufio"
 	"bytes"
 	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
-	"path"
 	"path/filepath"
 	"strings"
 	"time"
 
 	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/security/apparmor"
```

Comment:
I thought dependencies from test images into k8s.io/kubernetes were not desired?

Reply:
@liggitt Thanks for sharing that. If so, can we just revert the change and clean the TODO?

```
// TODO: This is copied from k8s.io/kubernetes/pkg/security/apparmor.getLoadedProfiles.
//       Refactor that method to expose it in a reusable way, and delete this version.
```